extension/js/common/core/mime.ts

```diff
@@ -156,7 +157,10 @@ export class Mime {
             } else if (Mime.getNodeType(node) === 'text/plain' && !Mime.getNodeFilename(node)) {
               mimeContent.text = Mime.getNodeContentAsUtfStr(node);
             } else if (Mime.getNodeType(node) === 'text/rfc822-headers') {
-              // todo - surface and render encrypted headers
+              const subjectsLines = Mime.getNodeContentAsUtfStr(node).match(/(?<=Subject:\s).*/gi);
```

Comment:
There should be `node.headers` property or something similar. Maybe `node.headers.Subject[0]`

Reply:
No, unfortunately,  there isn't the subject header. There are only 'Content-Type' and 'Content-Disposition headers'.

Reply:
But I found it in _parentNode. 
![image](https://user-images.githubusercontent.com/49104848/60369418-4500cf00-99fc-11e9-8a1b-0bed5ebead74.png)

I will try to access to it